app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    cp, old_file, new_file = command.split()
+
+    if cp != "cp":
```

Comment:
Code Efficiency Issue: The code checks if the first part of the command is 'cp', but it does not check if the command actually has three parts. You should add a check for this.